src/Globe/TileMesh.js

```diff
@@ -29,9 +29,11 @@ import GlobeDepthMaterial from 'Renderer/GlobeDepthMaterial';
 import MatteIdsMaterial from 'Renderer/MatteIdsMaterial';
 import RendererConstant from 'Renderer/RendererConstant';
 
-var groupelevation = [14, 11, 7, 3];
-var l_ELEVATION = 0;
-var l_COLOR = 1;
+const groupelevation = [14, 11, 7, 3];
+const l_ELEVATION = 0;
+const l_COLOR = 1;
+
+const NO_ELEVATION = -2;
```

Comment:
``` javascript
const NO_ELEVATION = -2;
```
Texture's Elevation has null, it's not sufficient to check if there's no elevation?